app/viewmodels/SummaryCardHelper.scala

```diff
@@ -179,7 +179,12 @@ class SummaryCardHelper @Inject()(link: views.html.components.link) extends Impl
           )
         )
       ),
-      summaryListRow(messages("summaryCard.lpp.key2"), Html(messages("summaryCard.lpp.15days")))
+      if(lpp.`type` == PenaltyTypeEnum.Additional){
```

Comment:
Should be a match statement on the reason enum that could be in it's own function that is called to get the reason key